infrastructure/engine/turbine_interface_bq.hh

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <infrastructure/balsa_queue/balsa_queue.hh>
+
+#include "infrastructure/engine/jetcat_turbine.hh"
+
+namespace jet {
+
+class TurbineInterfaceBQ : public BalsaQ {
+ public:
+  TurbineInterfaceBQ();
+  void init(int argc, char *argv[]);
+  void loop();
+  void shutdown();
+
+  void start_turbine();
+  void shutdown_turbine();
+  void set_turbine_throttle(uint32_t throttle_percent);
```

Comment:
[nit] Consider making these private -- even though no one will call this as a library